prometheus/exporter/gpu_exporter.go

```diff
@@ -166,12 +167,10 @@ func main() {
         log.Print("no gpu")   
         return   
     }
-    var index = 1
  
-    for a := 0; a < 1; {
+    for ; ; {
```

Comment:
> ; ; [](start = 7, length = 4)

remove

Reply:
[https://www.cyberciti.biz/faq/golang-for-loop-examples/](https://www.cyberciti.biz/faq/golang-for-loop-examples/), search for "Infinite for loop syntax"

---
In reply to: [160069097](https://github.com/Microsoft/pai/pull/188#discussion_r160069097) [](ancestors = 160069097)